migrations/20221011041400_add_mfa_indexes.up.sql

```diff
@@ -1,8 +1,18 @@
-create index if not exists refresh_token_session_id on {{ index .Options "Namespace" }}.refresh_tokens using btree(session_id);
-
 alter table {{ index .Options "Namespace" }}.mfa_amr_claims
-  add column if not exists id uuid not null,
-  add constraint amr_id_pk primary key(id);
+  add column if not exists id uuid not null;
+
+do language plpgsql
```

Comment:
nit: hmm is `plpgsql` really necessary? we don't seem to use any procedural elements or variables in the function so we can actually just use plain sql for it